### ISSUE TITLE
Bug Report - Update the broken link in the Footer

### ISSUE THREAD

**Issue by salman-farooqi** [PalisadoesFoundation/talawa-docs#638]
**Describe the bug**
Currently, the link is broken for the Footer's 'News' section, and is redirecting to an [old](https://docs.talawa.io/docs/features/news%20feed/news-feed-overview) URL .
I had updated the link in [this](https://github.com/PalisadoesFoundation/talawa-docs/pull/578) PR, but somehow its still showing the old one.

**To Reproduce**

1. Go to docusaurus.config.js in the talawa-docs files.
2. Update the link with this one (https://docs.talawa.io/docs/features/news-feed) in the code.

**Expected behavior**
![updated](https://user-images.githubusercontent.com/127323880/232136664-c20ec2c3-a2d6-40a5-94c0-a2711015010a.JPG)

**Actual behavior**
![the link address is broken in the footer for 'News'](https://user-images.githubusercontent.com/127323880/232136695-82771430-c21c-411a-bf75-f0f47e8c096c.JPG)

**Screenshots**
See Above.

**Additional details**
N/A

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by salman-farooqi** [comment]
Kindly assign, so I fix this.

**Comment by palisadoes** [comment]
Thanks for noticing this. 

1. We have stopped accepting changes to the footer as we are much more concerned with adding new content. 
2. However this link was working before.
3. To remove the risk of the footer being updated by accident again I have added the `docusaurus.config.js` file to the unauthorized list of files that will cause PRs to fail. 
4. I have also made the footer update to the correct location and verified the remaining links there.

In the past two months we have merged 14 PRs related to footer updates and it was becoming unmanageable.

Closing.
